src/components/Auth/Auth.js

```diff
@@ -0,0 +1,95 @@
+import { h, Component } from 'preact'
+import { AuthCheckProcessor } from './AuthCheckProcessor'
+import { FaceTecSDK } from '../../../core-sdk/FaceTecSDK.js/FaceTecSDK'
+import { Config } from './AuthConfig'
+import { FaceTecStrings } from './assets/FaceTecStrings'
+import { Button } from '@onfido/castor-react'
+
+export default class AuthCapture extends Component {
+  state = {
+    authConfig: {},
+    sessionInit: false,
+    sessionState: 'Initializing...',
+  }
+  componentDidUpdate(prevProps, prevState) {
+    if (
+      Object.keys(prevState.authConfig).length === 0 &&
+      Object.keys(this.state.authConfig).length > 0 &&
+      FaceTecSDK.getStatus() === 0
+    ) {
+      this.initFaceTec()
+    }
+  }
+
+  initFaceTec = () => {
+    FaceTecSDK.setResourceDirectory('../../../core-sdk/FaceTecSDK.js/resources')
+    FaceTecSDK.setImagesDirectory('../../../core-sdk/FaceTec_images')
```

Comment:
You can create an alias in `webpack.config.babel.js` so you can reference core-sdk much more easily by doing something like `'~/core-sdk/FaceTecSDK.js/resources'`